pytorch_lightning/core/lightning.py

```diff
@@ -1262,7 +1262,7 @@ def configure_optimizers(self):
                 # `scheduler.step()`. 1 corresponds to updating the learning
```

Comment:
It might be worth splitting this docstring now that this adds a hook for the scheduler logic specifically.

Reply:
well, frequency is still a part of `configure_optimizers`. the hook is used to configure how to do the `scheduler.step`, similar to `optimzer_step`. when to update is still decided internally using the frequency specified by the user.